### PR TITLE
Easy sections 613-648

### PR DESCRIPTION
**Pull request by Oliver-was-taken**
### Why?

All currently implementable sections from 613-648
prettier :)

### How?

add sections to yaml

## Anything else?

closes: #102
closes: #105 
closes: #111 
closes: #113 
closes: #122 
closes: #124 
closes: #135 
closes: #146 
closes: #225 
closes: #507 
closes: #508 
closes: #509
closes: #511 
closes: #512 
closes: #513 
closes: #514 
closes: #515 
closes: #517 
closes: #518

### CODE REVIEW

**Review by MagischerWolf05 — APPROVED** [review]
sounds Greate

File: fabbwled-backend/src/main/resources/sections/613-648.yaml [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+# Section 614 - too complex for the current implementation of sections
+# Section 615 - too complex for the current implementation of sections
+# Section 616 - too complex for the current implementation of sections
+- number: 617
+  content:
+    - text: |
+        A desperate battle ensues.
+        Ghoul, COMBAT 3, Defence 7, Stamina 15
+        If you win, turn to 196.
+        If you lose, your adventures are over, unless you have a resurrection deal.
```
Comment (Nilstrieb) [inline fabbwled-backend/src/main/resources/sections/613-648.yaml:11]: this should like, actually contain a fight. comment it out as too complex

File: fabbwled-backend/src/main/resources/sections/613-648.yaml [hunk 2]
```diff
@@ -0,0 +1,370 @@
...
+- number: 618
+  content:
+    - text: |
+        Becoming an initiate of Lacuna gives you the benefit of paying
+        less for blessings and other services the temple can offer. It costs
+        30 Shards to become an initiate. You cannot do this if you are
+        already an initiate of another temple. If you choose to become
+        an initiate, write Lacuna in the God box on your Adventure
+        Sheet – and remember to cross off the 30 Shards. Once you
+        have finished here, turn to 544.
```
Comment (Nilstrieb) [inline fabbwled-backend/src/main/resources/sections/613-648.yaml:22]: this is an if, and cannot be represented like that. also the god

File: fabbwled-backend/src/main/resources/sections/613-648.yaml [hunk 3]
```diff
@@ -0,0 +1,370 @@
...
+        The last time you were here, you had only a few minutes to
+        grab some loot. Desperately, you search around for the magic
+        chest that Oliphard the Wizardly wanted you to get for him.
+        You find it – note the magic chest on your Adventure Sheet –
+        but the sea dragon returns almost straight away, and you are
+        forced to climb out of the hole in the roof without getting
+        anything more, save for a pouch of 50 Shards.
+        You crawl out on to the island in the middle of the lake and
+        hitch a boat ride back to Cadmium. During the journey, you
+        examine the magic chest , but it is guarded by great sorcery and
```
Comment (Nilstrieb) [inline fabbwled-backend/src/main/resources/sections/613-648.yaml:85]:
```suggestion
        examine the magic chest, but it is guarded by great sorcery and
```

File: fabbwled-backend/src/main/resources/sections/613-648.yaml [hunk 4]
```diff
@@ -0,0 +1,370 @@
...
+        is impossible to open.
+        Delete the codeword Avenge and turn to 135 .
+    - turnTo: 135
+    - acquireKeyword: Avenge
```
Comment (Nilstrieb) [inline fabbwled-backend/src/main/resources/sections/613-648.yaml]: that's not acquire, it should be deleted

File: fabbwled-backend/src/main/resources/application.yaml [hunk 5]
```diff
@@ -23,4 +23,4 @@ spring:
       enabled: ALWAYS
 springdoc:
   api-docs:
-    path: "/api-docs"
\ No newline at end of file
+    path: "/api-docs"
```
Comment (keykey7) [inline fabbwled-backend/src/main/resources/application.yaml:26]: note that the current `.editorconfig` states `insert_final_newline = false`
while I am not a fan of this setting, that's currently the contract. if you deliberately want to update it, also reflect it in the `.editorconfig` file.

File: fabbwled-backend/src/main/resources/sections/613-648.yaml [hunk 6]
```diff
@@ -0,0 +1,370 @@
...
+    - text: |
+        You clamber down a ladder into a long, low hall, the Venefax
+        Market.
+        Items with no purchase price listed are not available locally.
+        Armour To buy To sell
+        Leather (Defence +1) 50 Shards 45 Shards
+        Ring mail (Defence +2) – 90 Shards
+        Weapons (sword, axe, etc) To buy To sell
+        Without COMBAT bonus 50 Shards 40 Shards
+        COMBAT bonus +1 – 200 Shards
```
Comment (Nilstrieb) [inline fabbwled-backend/src/main/resources/sections/613-648.yaml:101]: needs a marketplace, make it TODO

File: fabbwled-backend/src/main/resources/sections/613-648.yaml [hunk 7]
```diff
@@ -0,0 +1,370 @@
...
+          then:
+            - turnTo: 220
+- number: 625
+  content:
+    - text: |
+        You realize that you have committed an act of sacrilege by
+        stealing from the temple – you know in your heart that Tyrnai
+        has cursed you.
+        You will never be a great fighter while you suffer the wrath
+        of Tyrnai. Subtract 1 from your COMBAT ability until you can
```
Comment (Nilstrieb) [inline fabbwled-backend/src/main/resources/sections/613-648.yaml:134]: can't do this, TODO

File: fabbwled-backend/src/main/resources/sections/613-648.yaml [hunk 8]
```diff
@@ -0,0 +1,370 @@
...
+  content:
+    - text: |
+        You recite a devotional prayer to the gods, calling on your faith
+        to aid you. The goblin folk are repelled by your piety, and
+        purity of spirit. Even Gobrash finds the strength to get out from
+        under you, and run off, such is your effect on them.
+        ‘Eeaurgh!’ snarls the queen, recoiling, ‘Please, your godliness
+        is harmful to us. Here, take this and leave!’
+        A jug of faery mead and an enchanted spear (COMBAT
+        +2) are pushed towards you. Note them on your Adventure
```
Comment (Nilstrieb) [inline fabbwled-backend/src/main/resources/sections/613-648.yaml:149]: note them on the adventure sheet - cant do this

**Review by Nilstrieb — CHANGES_REQUESTED** [review]
Haven't reviewed the second half yet, but some feedback on the first half

**Review by lucasmillanm — APPROVED** [review]
Seems alright

**Review by Lion-alt — APPROVED** [review]
LGTM 👍  A test should be writen to catch those syntax errors...

File: fabbwled-backend/src/main/resources/sections/613-648.yaml [hunk 9]
```diff
@@ -0,0 +1,370 @@
...
+# Section 629 - too complex for the current implementation of sections
+- number: 630
+  content:
+    - text: |
+        You struggle deeper into the forest until you come to a thick
+        wall of impenetrable thorn bushes. Circling it, you find there is a
+        break in the hedge, but it is filled by a large tree.
+        To your surprise, a face forms in the trunk, and speaks in a
+        woody voice, ‘None can pass – begone, human!’
+        If you have the codeword Apple , turn to 594 . If not, you
```
Comment (Noeell) [inline fabbwled-backend/src/main/resources/sections/613-648.yaml:194]:
```suggestion
        If you have the codeword Apple , turn to 594 . If not, you can:
```

Maybe add can: here?

File: fabbwled-backend/src/main/resources/sections/613-648.yaml [hunk 10]
```diff
@@ -0,0 +1,370 @@
...
+- number: 641
+  content:
+    - text: |
+        Want to try your luck in the Gambler’s Den?’ asks a short, dark
+        man, dressed as a mercenary bodyguard. ‘Just 5 Shards entrance
+        fee.’
+        He nods to a door at the top of some rickety stairs.
+        If you pay the money, cross it off and turn to 91 . If not, he
+        shrugs, and hisses at another passerby. You wander off and pass
+        an uneventful evening, turn to 100 .
```
Comment (Noeell) [inline fabbwled-backend/src/main/resources/sections/613-648.yaml:299]:
```suggestion
        an uneventful evening, turn to 100.
```

File: fabbwled-backend/src/main/resources/sections/613-648.yaml [hunk 11]
```diff
@@ -0,0 +1,370 @@
...
+        You open the cage door. A wide grin suddenly appears at about
+        head height within the dark shadows that wreath the trau. All
+        you can see is a huge sliver of a grin, topped by two glowing red
+        eyes.
+        ‘Free at last!’ gushes the trau as it leaps out. ‘Now for home,
+        and as much faery mead as I can drink!’
+        With that, it delves into the ground so fast that it has
+        burrowed out of sight before you can do anything to stop it.
+        Shrugging your shoulders resignedly, you leave the market.
+        Turn to 400 .
```
Comment (Noeell) [inline fabbwled-backend/src/main/resources/sections/613-648.yaml:320]:
```suggestion
        Turn to 400.
```

File: fabbwled-backend/src/main/resources/sections/613-648.yaml [hunk 12]
```diff
@@ -0,0 +1,370 @@
...
+          then:
+            - turnTo: 78
+# Section 445 - too complex for the current implementation of sections
+- number: 644
+  content:
+    - text: |
+        Using your sorcerous powers you breathe a cloud of greenish
+        vapour over the ratmen. Coughing and gasping, they sink into
+        an enchanted slumber. It is an easy matter to despatch them
+        while they sleep. Turn to 554 .
```
Comment (Noeell) [inline fabbwled-backend/src/main/resources/sections/613-648.yaml:343]:
```suggestion
        while they sleep. Turn to 554.
```

File: fabbwled-backend/src/main/resources/sections/613-648.yaml [hunk 13]
```diff
@@ -0,0 +1,370 @@
...
+    - turnTo: 554
+- number: 646
+  content:
+    - text: |
+        Your mind is blasted into numb confusion by the protective
+        spells that guard the sacred grove. You wander off aimlessly into
+        the forest, and get hopelessly lost for days. You don’t even have
+        the wit to eat and drink – lose 3 Stamina points. If you are still
+        alive, you wander through the forest until you emerge, at last, at
+        the Bronze Hills. Turn to 110 .
```
Comment (Noeell) [inline fabbwled-backend/src/main/resources/sections/613-648.yaml:353]:
```suggestion
        the Bronze Hills. Turn to 110.
```

File: fabbwled-backend/src/main/resources/sections/613-648.yaml [hunk 14]
```diff
@@ -0,0 +1,370 @@
...
+    - text: |
+        A bard in the tavern recites an epic tale of ancient legend. He
+        speaks of Xinoc the Priest King, the ruler of a mighty nation in
+        the distant steppes, far to the north. A thousand years ago his
+        empire flourished, and his wealth was enormous. He was buried
+        in a great pyramid tomb, along with his riches. It is said the
+        Pyramid of Xinoc lies beyond the Spine of Harkun, and beyond
+        the Great Steppes, in the foothills of the Peaks at the Edge of the
+        World. You question him further, but that is all he knows. You
+        leave the tavern. Turn to 10
```
Comment (Noeell) [inline fabbwled-backend/src/main/resources/sections/613-648.yaml:182]:
```suggestion
        leave the tavern. Turn to 10.
```

File: fabbwled-backend/src/main/resources/sections/613-648.yaml [hunk 15]
```diff
@@ -0,0 +1,370 @@
...
+  content:
+    - text: |
+        You realize that you have committed an act of sacrilege by
+        stealing from the temple – you know in your heart that Tyrnai
+        has cursed you.
+        You will never be a great fighter while you suffer the wrath
+        of Tyrnai. Subtract 1 from your COMBAT ability until you can
+        find a way of lifting the curse. Perhaps the priests of Sig in
+        Marlock City can help you. Note that you are under ‘Tyrnai’s
+        Curse, -1 COMBAT. You sneak out of the temple. Turn to 400 .
```
Comment (Noeell) [inline fabbwled-backend/src/main/resources/sections/613-648.yaml:137]:
```suggestion
        Curse, -1 COMBAT. You sneak out of the temple. Turn to 400.
```

File: fabbwled-backend/src/main/resources/sections/613-648.yaml [hunk 16]
```diff
@@ -0,0 +1,370 @@
...
+  content:
+    - text: |
+        To renounce the worship of Alvir and Valmir, you must pay 30
+        Shards in compensation to the priesthood. The priest says
+        nothing, he just points to a mural on the wall. It depicts the
+        story of a woman who left the temple – a storm destroyed her
+        ship and she drowned, to spend eternity as a lost soul, cleaning
+        the barnacles off the thrones of Alvir and Valmir.
+        Do you want to change your mind?
+        When you have finished here, turn to 220 .
```
Comment (Noeell) [inline fabbwled-backend/src/main/resources/sections/613-648.yaml:118]:
```suggestion
        When you have finished here, turn to 220.
```

File: fabbwled-backend/src/main/resources/sections/613-648.yaml [hunk 17]
```diff
@@ -0,0 +1,370 @@
...
+        chest that Oliphard the Wizardly wanted you to get for him.
+        You find it – note the magic chest on your Adventure Sheet –
+        but the sea dragon returns almost straight away, and you are
+        forced to climb out of the hole in the roof without getting
+        anything more, save for a pouch of 50 Shards.
+        You crawl out on to the island in the middle of the lake and
+        hitch a boat ride back to Cadmium. During the journey, you
+        examine the magic chest, but it is guarded by great sorcery and
+        is impossible to open.
+        Delete the codeword Avenge and turn to 135 .
```
Comment (Noeell) [inline fabbwled-backend/src/main/resources/sections/613-648.yaml:87]:
```suggestion
        Delete the codeword Avenge and turn to 135.
```

**Review by Noeell — CHANGES_REQUESTED** [review]
Please link section-issues

**Review by Noeell — APPROVED** [review]
LGTM 👍

**Review by Nilstrieb — APPROVED** [review]

**Pull request merged**